src/AmbientSounds.Uwp/Views/ScreensaverPage.xaml.cs

```diff
@@ -232,5 +245,39 @@ private void OnToggleFullscreen(object sender, RoutedEventArgs e)
                 });
             }
         }
+
+        private void SetTimer()
+        {
+            InactiveTimer = Queue.CreateTimer();
+            InactiveTimer.Interval = new TimeSpan(0, 0, SecondsToHide);
+            InactiveTimer.IsRepeating = false;
+            InactiveTimer.Tick += OnInactive;
+        }
+
+        private void OnInactive(DispatcherQueueTimer t, object sender)
+        {
+            if (!IsButtonsHidden)
+            {
+
+                GoBackButton.Visibility = Visibility.Collapsed;
+                ActionButtons.Visibility = Visibility.Collapsed;
+                IsButtonsHidden = true;
+
+            }
+
+            InactiveTimer.Stop();
+        }
+
+        private void RootPage_OnPointerMoved(object sender, PointerRoutedEventArgs e)
+        {
+            if (IsButtonsHidden)
+            {
+                GoBackButton.Visibility = Visibility.Visible;
+                ActionButtons.Visibility = Visibility.Visible;
+                IsButtonsHidden = false;
+            }
+
+            InactiveTimer.Start();
+        }
```

Comment:
The [Debounce Toolkit Helper](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.dispatcherqueuetimerextensions.debounce) would probably work for this:

```suggestion
        private void RootPage_OnPointerMoved(object sender, PointerRoutedEventArgs e)
        {
            AreButtonsVisible = true;
            
            _inactiveTimer.Debounce(() => { AreButtonsVisible = false; }, TimeSpan.FromSeconds(SecondsToHide));
        }
```